src/components/error-message/error-message.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { css } from 'linaria'
+
+const errorMessageStyles = css`
+  color: red;
```

Comment:
Should we use one of the color variables here?

Reply:
Good call! My placeholder got away from me.